src/db.py

```diff
@@ -741,6 +812,10 @@ def _upgrade(oldversion):
                 # player id as a string). When nocasing players, this may cause some records to be merged.
                 with open(os.path.join(dn, "db", "upgrade2.sql"), "rt") as f:
                     c.executescript(f.read())
+            if oldversion < 3:
+                print ("Upgrade from version 2 to 3...", file=sys.stderr)
+                with open(os.path.join(dn, "db", "upgrade3.sql"), "rt") as f:
+                    c.executescript(f.read())
```

Comment:
There is no guarantee that a version upgrade will consist of a simple sql script to execute; there may be python code needed or for things to be broken out in a different way.